lib/src/rows/types.tsx

```diff
@@ -0,0 +1,42 @@
+export type RowsProps = {
+  /**
+   * Align columns in the main axis.
+   */
+  alignX?: "start" | "end" | "center" | "baseline" | "stretch";
+  /**
+   * Align columns in the cross axis.
+   */
+  alignY?: "start" | "end" | "center" | "baseline" | "stretch";
+  /**
+   * Set the component to a different HTML tag.
+   */
+  as?: React.ElementType;
+  /**
+   * Add a divider between rows.
+   */
+  divider?: boolean;
+  /**
+   * Spacing between rows.
+   */
+  gutter?: "0rem" | "0.125rem" | "0.25rem" | "0.5rem" | "0.75rem" | "1rem" | "1.5rem" | "2rem";
+  /**
+   * Change the flex-direction of the row to reverse.
+   */
+  reverse?: boolean;
+  /**
+   * Single Row.
+   */
+  children: React.ReactElement[] | React.ReactElement;
+};
+
+export type RowProps = {
+  /**
+   * Fixed width of the single row.
+   */
+  height?: "auto" | "content" | "1" | "2" | "3" | "4" | "5" | "6" | "7" | "8";
+  /**
+   * Custom content inside the row.
+   */
+  children: React.ReactNode;
+  alignY?: "start" | "end" | "center" | "baseline" | "stretch";
```

Comment:
As we are passing `alignY` as a prop, anyone from the outside can use it, both in Rows and Row. We need to pass the value from father to child by using a context to avoid external usage.

![image](https://user-images.githubusercontent.com/44321109/175943878-3e9d9eee-c912-4078-8c5b-0a75488264da.png)

Reply:
Done